### ISSUE TITLE
Has Apple been notified?

### ISSUE THREAD

**Issue by NiklasBr** [Pr0x13/iDict#1]
Just a question here. Has Apple been notified of this in advance? And if so, how long time were they given to respond?

[Product Security](https://www.apple.com/support/security/): To report security or privacy issues that affect Apple products or web servers, please contact: product-security@apple.com.

**Comment by Gyvastis** [comment]
Apple now blocks the account after 5 or 6 incorrect requests now.

**Comment by NiklasBr** [comment]
Yes, it seems Apple reacted within hours of this repository being posted.

But that's not really what I asked about. I wondered if the author or someone else had contacted Apple. I.e. had some method of [Responsible disclosure](https://en.wikipedia.org/wiki/Responsible_disclosure) been used to alert prior to posting this to the public?
